projects/admin/src/app/record/detail-view/library-detail-view/library-detail-view.component.html

```diff
@@ -16,15 +16,15 @@
 -->
 <ng-container *ngIf="record$ | async as record">
   <h1>{{ record.metadata.name }}</h1>
-  <section class="row">
+  <dl class="row mb-3">
     <!-- ADDRESS -->
     <ng-container *ngIf="record.metadata.address">
-        <dt class="col-sm-3 offset-sm-2 offset-md-0">
-            {{ 'Address' | translate }}:
-          </dt>
-          <dd class="col-sm-7 col-md-8 mb-0">
-            {{ record.metadata.address }}
-          </dd>
+      <dt class="col-sm-3 offset-sm-2 offset-md-0">
+        {{ 'Address' | translate }}:
+      </dt>
```

Comment:
```
<dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>
  Address
</dt>
```

Reply:
Great. I didn't now that this exists. Needs refactor other files in a separate PR.